src/floor/RequestSimulator.java

```diff
@@ -0,0 +1,91 @@
+package floor;
+
+import java.io.IOException;
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.SocketException;
+
+import messages.ElevatorRequestMessage;
+import messages.ElevatorRequestMessage.Direction;
+import messages.Message;
+
+public class RequestSimulator implements Runnable{
+	
+	DatagramSocket sendSocket;
+	DatagramPacket sendPacket;
```

Comment:
Nitpick: Instead of using an ivar sendPacket, initialize it here and use the Message.send